release-notes/6.0/preview/6.0.0-rc.1.md

```diff
@@ -36,6 +36,12 @@ You can check your .NET SDK version by running the following command. The exampl
 $ dotnet --version
 6.0.100-rc.1.21458.32
 ```
+### .NET 6.0 RC1 Support and Servicing
+ 
+.NET 6 RC1 is supported for use in production, which means you can build an app using .NET 6 RC1, deploy this app to a production environment, and expect the level of support and servicing you would normally get for a shipped release such as .NET 5.0. We will continue to support RC1 for 30 days after RC2 ships and support RC2 for 30 days after .NET 6.0 GA ships.
+ 
+Any bug fixes including security fixes for RC1 and beyond will be handled similar to any other shipped product version like .NET 5.0. These servicing fixes will be made available as a monthly rollup in the next successive RC or GA release i.e. fixes for RC1 will be rolled up into RC2, and fixes for RC2 will be rolled up into .NET 6.0 GA.
```

Comment:
```suggestion
.NET 6 RC1 will not be updated after it is released. You will need to update your environment to .NET 6 RC2 and the final .NET 6 stable release for future improvements, including security updates.
```

Reply:
Building on to of @richlander's suggestion:
```suggestion
.NET 6 RC1 will not be updated after it is released. You will need to update your environment to .NET 6 RC2 and .NET 6 GA for future improvements, including security updates.
```